### DISCUSSION TITLE
Auto Fix warnings?

### DISCUSSION THREAD

**Discussion by TinyTimothy2** [TabularEditor/TabularEditor3#1236] [🙋♀️TE3 Q&A] [answered]
Is there a way to autofix warnings.  I have some older models where i didn't qualify col in measure.  TE sugguest the appropriate fix.
I can't seem to select a batch and choose 'fix'

**Comment by otykier** [comment, marked as answer]
Formatting the code (F6) should ensure that columns are qualified and measures aren't, as long as you have this setting enabled under **Tools > Preferences > DAX Editor > Auto Formatting**:
<img width="701" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/8976200/521e433b-0612-405b-9437-4ec35792e0f8">
You can select multiple measures or even entire tables and use the right-click > Script DAX option, to generate a document containing all of their DAX expressions, so you can format them all at once.
